app/views/browse/build.html

```diff
@@ -123,9 +123,9 @@
                         <span class="space-after">{{build.status.phase}}</span>
 
                         <div ng-if="build.status.startTimestamp && !logEmpty" class="log-timestamps">
-                          Log from {{build.status.startTimestamp  | date : 'short'}}
+                          Log from {{build.status.startTimestamp  | date : 'MMM dd, yyyy h:mm a'}}
```

Comment:
Should we use `date : 'medium'` which is the almost the same, but will be localized?

https://docs.angularjs.org/api/ng/filter/date